jobs/search-alert/search_alert/main.py

```diff
@@ -0,0 +1,337 @@
+import click
+import datetime as dt
+import holidays
+import numpy as np
+import pandas as pd
+from google.cloud import bigquery
+
+
+# check whether the day +/- 1 day is holiday
+def is_it_holiday(ds, country):
+	isHoliday = False
+	ds_range = [pd.to_datetime(ds) + dt.timedelta(days = x) for x in range(-2,2,1)] # get clever about the holiday if it's close to weekend
+	try:
+		isHoliday_range = [pd.to_datetime(ds) in holidays.CountryHoliday(country) for ds in ds_range]
+		isHoliday = max(isHoliday_range)
+	except:
+		pass
+	if country == 'CN':
+		isHoliday = max([True if (x.month == 10) & (x.day == 1) else False for x in ds_range])
+	if isHoliday == False:
+		isHoliday = max([True if (x.month == 12) & (x.day >= 25) else False for x in ds_range]) # Christmax
+	if isHoliday == False:
+		isHoliday = max([True if (x.month == 1) & (x.day == 1) else False for x in ds_range]) # NewYear
+	if isHoliday == False:
+		isHoliday = max([True if (x.month == 5) & (x.day == 1) else False for x in ds_range]) # National Labor Day
+	return isHoliday
+
+
+def get_days_since_1970(adate):
+   return (pd.to_datetime(adate)-pd.to_datetime(1970,1,1)).dt.days
+
+
+@click.command()
+@click.option("--submission_date", required=True)
+@click.option('--dry_run', is_flag=True, default=False)
+def main(submission_date, dry_run):
+    project_id = 'mozdata'
```

Comment:
project_id could be a constant on top of the module or a main() param?

Reply:
+1, I'll make this a parameter.